app/views/application/_header.html.erb

```diff
@@ -1,8 +1,7 @@
 <header class="site-header">
-  <nav class="navbar navbar-expand-md navbar-light bg-light">
+  <nav class="navbar navbar-expand-md">
     <a class="navbar-brand" href="/">
       <%= image_tag(asset_pack_path('media/images/logo.png'), class: 'logo d-inline-block align-top', alt: 'Soul Medicine Logo') %>
-      Soul Medicine
```

Comment:
For SEO purposes, may still be good to have this, but hidden? You can use [the `sr-only` class](https://getbootstrap.com/docs/4.3/utilities/screen-readers/) or [the `invisible` class](https://getbootstrap.com/docs/4.3/utilities/visibility/).